dumpling/tests/all_generate_column/conf/diff_config.toml

```diff
@@ -1,40 +1,28 @@
 # diff Configuration.
 
-log-level = "info"
-
-chunk-size = 1000
-
 check-thread-count = 4
 
-sample-percent = 100
+export-fix-sql = true
 
-use-checksum = true
+check-struct-only = false
 
-fix-sql-file = "fix.sql"
+[task]
+    output-dir = "./output"
```

Comment:
Can we print the result in dumpling's test dir `$DUMPLING_OUTPUT_DIR`? To make it easier to track in the future.